### PR TITLE
Update badges and mention links colors

### PR DESCRIPTION
**Pull request by tassoevan**
Captura de tela de 2019-04-12 17-59-15

### CODE REVIEW

**Comment by rodrigok** [comment]
@tassoevan is this ready to be reviewed?

**Comment by tassoevan** [comment]
@rodrigok Previous commits were broken into single PRs for faster code review, and I want to review the new color palette with @thiagosanchz before considering it ready to merge.

File: app/mentions/client/mentionLink.css [hunk 1]
```diff
@@ -1,27 +1,37 @@
+.message .mention-link,
 .mention-link {
 	padding: 0 6px 2px;
 
 	transition: border-radius 0.3s;
```
Comment (sampaiodiego) [inline app/mentions/client/mentionLink.css]: should this transition be changed to `opacity` or removed at all?

File: app/ui/client/views/app/room.js [hunk 2]
```diff
@@ -931,6 +950,7 @@ Template.room.onCreated(function() {
 	this.flexTemplate = new ReactiveVar;
 
 	this.userDetail = new ReactiveVar(FlowRouter.getParam('username'));
+	this.groupDetail = new ReactiveVar('all');
```
Comment (sampaiodiego) [inline app/ui/client/views/app/room.js]: this changed the default member list options from `Online` to `All`.. is that intentional?

Comment (tassoevan) [reply app/ui/client/views/app/room.js]: It wasn't, I'll patch it.

**Pull request merged**